docs/jvm_native_images.md

```diff
@@ -0,0 +1,18 @@
+# Native images for the Java Virtual Machine (JVM)
```

Comment:
nit: add GraalVM to the start of this header to avoid confusion.

Reply:
Ah - I was referring to native images in general - not just those of the GraalVM flavour. Right now, Android produces some native-esk (right?), and then there's Scala Native and perhaps others in future.

Reply:
OK, lets title it "Ahead of Time (AOT) compilation for the Java Virtual Machine (JVM)". This will represent that intention more generically I think. Also, renaming the doc to match jvm_aot.md or something of that nature?

Reply:
Done

Reply:
Thank you!